datahub-web-react/src/app/entity/shared/containers/profile/sidebar/Domain/SetDomainModal.tsx

```diff
@@ -48,35 +42,53 @@ export const SetDomainModal = ({ visible, onClose, refetch }: Props) => {
 
     const inputEl = useRef(null);
 
-    const onOk = async () => {
-        if (!selectedDomain) {
-            return;
-        }
-        try {
-            await setDomainMutation({
+    useEffect(() => {
+        setTimeout(() => {
+            if (inputEl && inputEl.current) {
+                (inputEl.current as any).focus();
+            }
+        }, 1);
+    });
```

Comment:
same comment about `autoFocus` on the input not working?

Reply:
Removed and fixed it by unmounting the component.

```
            {showModal && (
                <SetDomainModal
                    visible={showModal}
                    refetch={refetch}
                    onCloseModal={() => {
                        setShowModal(false);
                    }}
                />
            )}
```